test/helpers/index.ts

```diff
@@ -205,6 +208,7 @@ export const batchMintAndListRemovalsForSale = async ({
         removalData: {
           supplierAddress: removalData.supplierAddress ?? supplier,
           vintage: removalData.vintage ?? defaultStartingVintage + index,
+          subIdentifier: generateRandomSubIdentifier(),
```

Comment:
love love love

Reply:
maybe it's time to introduce [faker](https://github.com/faker-js/faker)?